src/sentry/static/sentry/less/forms.less

```diff
@@ -154,12 +154,12 @@ label {
 }
 
 .has-error .form-control {
-  border-color: #c9c0d1;
+  border-color: #c9c0d1 !important;
```

Comment:
Why are we adding this? this could have unwanted effects in other places

Reply:
yes I know but at the moment if we don't add !important tag, some global selector is setting a red color that we don't use... it's wrong.